metricbeat/module/system/core/_meta/fields.yml

```diff
@@ -6,6 +6,7 @@
   fields:
     - name: id
       type: long
+      dimension: true
```

Comment:
this is an example where flagging the field as `dimension` was required. `id` should actually be a `keyword` here, but we may find other places.